kubectl-plugin/cmd/kubectl-ray.go

```diff
@@ -5,13 +5,15 @@ import (
 
 	cmd "github.com/ray-project/kuberay/kubectl-plugin/pkg/cmd"
 	flag "github.com/spf13/pflag"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
 )
 
 func main() {
 	flags := flag.NewFlagSet("kubectl-ray", flag.ExitOnError)
 	flag.CommandLine = flags
+	ioStreams := genericclioptions.IOStreams{In: os.Stdin, Out: os.Stdout, ErrOut: os.Stderr}
```

Comment:
`genericclioptions.IOStreams` seems to have already been deprecated https://pkg.go.dev/k8s.io/cli-runtime/pkg/genericclioptions#IOStreams.

Reply:
Changed it to use `genericiooptions.IOStreams` instead